src/java/org/apache/cassandra/service/StorageProxy.java

```diff
@@ -1220,12 +1261,12 @@ public static void mutateWithTriggers(List<? extends IMutation> mutations,
                               .viewManager
                               .updatesAffectView(mutations, true);
 
+
         long size = IMutation.dataSize(mutations);
         writeMetrics.mutationSize.update(size);
         writeMetricsForLevel(consistencyLevel).mutationSize.update(size);
-        NonSerialWriteStrategy nonSerialWriteStrategy = DatabaseDescriptor.getNonSerialWriteStrategy();
-        if (nonSerialWriteStrategy.writesThroughAccord && !SchemaConstants.getSystemKeyspaces().contains(keyspaceName))
-            mutateWithAccord(augmented != null ? augmented : mutations, consistencyLevel, queryStartNanoTime, nonSerialWriteStrategy);
+        if (writesThroughAccord(mutations) && !SchemaConstants.getSystemKeyspaces().contains(keyspaceName))
```

Comment:
This check for the system keyspaces is gone now. Should still be kept around as an assertion in case we have test code or otherwise that ends up here?